go.sum

```diff
@@ -1,135 +1,316 @@
-github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
why did this trigger so many changes in go.sum? go.mod seems more reasonable since there are roughly the same number of new lines as old lines.

Reply:
Good question. I'm not sure. They are from the Viper go.sum, but I don't understand why the hashes need to exist. go.sum is kind of a mystery to me.